src/components/Inbox/NewMessage.jsx

```diff
@@ -0,0 +1,175 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useSession } from '@inrupt/solid-ui-react';
+import { sendMessageTTL } from '../../utils';
+
+/**
+ * New Message Component - Component that allows user to write
+ *  a message to another user from their inbox
+ *
+ * @memberof Inbox
+ * @name NewMessage
+ */
+
+const NewMessage = () => {
+  const { session } = useSession();
+  const [message, setMessage] = useState({
+    senderName: '',
+    recipientName: '',
+    recipientUsername: '',
+    title: '',
+    message: ''
+  });
+  const [error, setError] = useState('');
+  const [success, setSuccess] = useState('');
+
+  // Modifies message upon input
+  const handleChange = (e) => {
+    setMessage({
+      ...message,
+      [e.target.id]: e.target.value
+    });
+  };
+
+  // Handles submit (awaiting functionality for this)
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    if (!message.title) {
+      setError('Please enter a value for Message Title');
+    } else if (!message.recipientUsername) {
+      setError('Please enter a value for Recipient Username');
+    } else if (!message.recipientName) {
+      setError('Please enter a value for Recipient Full Name');
+    } else if (!message.message) {
+      setError('Please enter a value for the Message');
+    } else if (!message.senderName) {
+      setError('Please enter a value for Sender Full Name');
+    } else {
+      try {
+        await sendMessageTTL(session, message);
+        setMessage({
+          senderName: '',
+          recipientName: '',
+          recipientUsername: '',
+          title: '',
+          message: ''
+        });
+        setError('');
+        setSuccess(`Message successfully sent to ${message.recipientUsername}`);
+      } catch (err) {
+        // TODO: Make sure invalid username is the only possible error
+        setError('Recipient username does not exist');
+      }
+    }
+  };
+
+  /* eslint-disable jsx-a11y/label-has-associated-control */
+  return (
+    <StyledForm onSubmit={(e) => handleSubmit(e)} autoComplete="off">
+      <StyledNotice>* indicates a required field</StyledNotice>
```

Comment:
All this about the `*` indicators and `required fields` etc, can be cleaned up later with MUI and a lot of it's built in styling, validation, color theme for error/success, helper texts... I think it will lead to less lines of code when done in future PR.